### DISCUSSION TITLE
🐛 BUG:not have auto completion for tag

### DISCUSSION THREAD

**Discussion by BingCoke** [withastro/language-tools#769] [Feature requests]
### Describe the Bug

as you can see,i try to autocompetion for Card tag,but get noting
![CopyQ fnaFbg](https://github.com/withastro/language-tools/assets/81607010/fa5e31da-3058-46c9-bb3a-aff4a2056c3b)
but if i have a complete tag, we will get a aucocompeltion for Card

![CopyQ qcGXkp](https://github.com/withastro/language-tools/assets/81607010/862fc472-ccab-42b4-991c-9f5f246a52d9)
btw,I use neovim to coding astro, config is 
```lua
local astrocap = require("cmp_nvim_lsp").default_capabilities()
lspconfig.astro.setup({
	capabilities = astrocap,
	on_attach = on_attach,
})

```

### Steps to Reproduce

1. `npm init astro` using template 
2. use a tag for auto completion
3.

**Comment by Princesseuh** [comment]
At the moment, this is an intentional choice because HTML (the tags completions) and TypeScript (the Component completions) completions otherwise enter in conflict when they both run at the same time, it's a bit annoying.

Moving this to discussion since it's more of a feature request.
